apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -50,15 +38,41 @@ export default class StartNewProject extends React.Component {
     const DEFAULT_PROJECT_TYPES_BASIC = [
       'spritelab',
       'artist',
-      'minecraft_designer',
       'dance',
-      'poetry'
+      'playlab'
     ];
 
     const defaultProjectTypes = canViewAdvancedTools
       ? DEFAULT_PROJECT_TYPES_ADVANCED
       : DEFAULT_PROJECT_TYPES_BASIC;
 
+    const OPEN_ENDED = ['spritelab', 'dance', 'poetry', 'thebadguys'];
+
+    const DRAWING = ['artist', 'frozen'];
```

Comment:
Just an idea: should we prefix/suffix all of these with something like `_PROJECT-TYPES`?